db/migrate/20160224011900_create_data_csvs.rb

```diff
@@ -0,0 +1,226 @@
+class CreateDataCsvs < ActiveRecord::Migration
+  def change
+    create_table :data_csvs do |t|
+      t.string :facility_code, null: false
+      t.string :institution, null: false
+      t.string :city
+      t.string :state
+      t.string :zip
+      t.string :country
+      t.string :va_highest_degree_offered
+      t.string :type # BEWARE!!!!
+      t.integer :bah
+      t.boolean :poe
```

Comment:
There is a data dictionary that comes with the Data Service that defines each field and sources the information.